itemtypes.md

```diff
@@ -0,0 +1,40 @@
+# Item Type Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint is currently still named entitytypes, but may be renamed still.
+
+It contains the various types of items (publication, person, journal, …)
+
+## Main Endpoint
+**/api/core/entitytypes**
```

Comment:
It looks like this PR has forked from #40?   Some of these changes are from #40, and this endpoint was renamed to `itemtypes`.

Reply:
I've merged the changes now